### ISSUE TITLE
Not an issue

### ISSUE THREAD

**Issue by cinnamoner** [cinnamoner/Clipboard#3]
@LestcapeThanks for posting that fix. Sorry I couldn't post there but I told the guy I'm not going to anymore. And what I say I must respect.

Visiting that thread was quite a disappointment for me. I've found out Mint suffers from yet another shortcoming: arrogance. I wanted to download Mint again but I'm not going to anymore. I'm not going to eat crap from Mint again. That thread is illustrative of how Clem, his dogs and his fanatics treat poeple acting in good faith. Mint hasn't progressed much lately because they don't need to. They run a business that flourishes and they don't care anymore. Not only are they arrogant, condescending and patronizing but they are greedy as well. Otherwise why would they whipe out a searchengine? Their position on Distrowatch might be an arrangement as well. I've seen distros which are way ahead of Mint and their leaders are modest and care about people. Mint used to be innovative and I credit them for their major contribution to the GNU/Linux desktop. Those times are gone. They are not worthy of my attention anymore and I sure won't be wasting my time arguing with them. That's my take on Linux Mint.

**Comment by lestcape** [comment]
@cinnamoner we can act with arrogance in some context and that's not mean we are arrogant... In that little world of free software be arrogant it's difficult... It's a high level competition to be the best in all things and be who know all things... See this thread: https://bugzilla.gnome.org/show_bug.cgi?id=778809 I stop the conversation there, because this will not go to any place... He only try to demonstrate how I'm wrong, not matter the issue it self...

Anyway, i agree that this is not the way to receive the user comments... I go out of Mint (cinnamon), for other reasons (i removed all my issues and pull request)... I just commented in the issue, because i dislike the way of how they are acting with you... But i really will not do anything else for cinnamon. Instead i will do a lot of things for my own thing... I fork cinnamon to be running at top of gnome shell, and i think this will take me a lot of time, but not matter... This will resolve all problems that i have and also will provide a way to create new things... You are invited to help in this enforces if you want and how you want/can... It's not time to detect bugs because have a lot, it's not time for create themes, or for anything else than make code and make things works, but the time where your help will be appreciated will come in the future if you want help on somethings...  I hope that this will work also better than in cinnamon in the future, but if not, not matter. It's really a good experience.

**Comment by cinnamoner** [comment]
@lestcape Thank you very much 👍

**Comment by cinnamoner** [comment]
@lestcape Yeah, I've been watching classic gnome for quite a while. It's an interesting project.
***
Speaking of gnome, seems Nautillius and its offshoots suffer from an annoying characteristic: the icons glow up on mouse hover. It's especially annoying when the folders are yellow as in Elbullazul's Windows themes.
***
There's another thing too: in Windows Explorer, there are some effects on mouse actions whilst in Nemo there aren't any.
***
So, basically what I don't like in Cinnamon is:
1. No Desktop Grid for icon alignment 
2. Icon glow-up on mouse hover 
3. Lack of effects in the file manager on mouse actions  (this is hard to explain. The best way to understand what I mean is to open Windows Explorer and to exercise the mouse over icons/folders)

**Comment by lestcape** [comment]
@cinnamoner ok, but you can use that applet: https://cinnamon-spices.linuxmint.com/applets/view/268

**Comment by cinnamoner** [comment]
I'm beholding it.

**Comment by cinnamoner** [comment]
@lotsrc

**Comment by cinnamoner** [comment]
@lestcape Yeah, seems a cool little applet. I've posted some questions. But you already know that. Thanks 👍

**Comment by cinnamoner** [comment]
@lestcape Felt the need to delete that post. I hope I haven't upset you.

**Comment by cinnamoner** [comment]
> I just verified that my modification still does the job. Unfortunately my pull request #696 was closed).

> For everybody desperately wanting this feature:
For debian/ubuntu/mint:

> get compiler and everything sudo apt-get install build-essential dpkg-dev
activate source repositories in package-sources (https://forums.linuxmint.com/viewtopic.php?t=206245)
get the nemo sourcecode apt-get source nemo
get everything needed to build nemo sudo apt-get build-dep nemo
modify source as described in t-oster@ae0262e cd nemo* && pluma libnemo-private/nemo-icon-container.c (just change DESKTOP_PAD_VERTICAL to 1 and SNAP_SIZE_Y to 88)
build the package dpkg-buildpackage -rfakeroot -us -uc
install it sudo dpkg -i ../nemo*.deb
open dconf-editor and change the setting /org/nemo/desktop/text-ellipsis-limit to 2 or just dconf write /org/nemo/desktop/text-ellipsis-limit 2
Open nemo settings and disable the preview
log out and in and enjoy

**Comment by cinnamoner** [comment]
I just verified that my modification still does the job. Unfortunately my pull request #696 was closed).

For everybody desperately wanting this feature:
For debian/ubuntu/mint:

get compiler and everything sudo apt-get install build-essential dpkg-dev
activate source repositories in package-sources (https://forums.linuxmint.com/viewtopic.php?t=206245)
get the nemo sourcecode apt-get source nemo
get everything needed to build nemo sudo apt-get build-dep nemo
modify source as described in t-oster@ae0262e cd nemo* && pluma libnemo-private/nemo-icon-container.c (just change DESKTOP_PAD_VERTICAL to 1 and SNAP_SIZE_Y to 88)
build the package dpkg-buildpackage -rfakeroot -us -uc
install it sudo dpkg -i ../nemo*.deb
open dconf-editor and change the setting /org/nemo/desktop/text-ellipsis-limit to 2 or just dconf write /org/nemo/desktop/text-ellipsis-limit 2
Open nemo settings and disable the preview
log out and in and enjoy

**Comment by cinnamoner** [comment]
@t-oster @feren Guys, you're welcome to discuss it right here.

**Comment by cinnamoner** [comment]
> KDE has it, XFCE (!!) has it being a super lightweight environment. Windows 98 had it and every release since, OSX has had it since the mid-90's. I can't get it into my head why any modern desktop environment (cough GNOME3, LXDE, Cinnamon, MATE, ……) wouldn't implement this.

Well said.

**Comment by cinnamoner** [comment]
@t-oster I wonder whether you would come over for a conversation. I'm not posting on that thread, because I said I would never post there again.

**Comment by cinnamoner** [comment]
https://github.com/lotsrc/DesktopIconSize/issues/1

**Comment by t-oster** [comment]
@cinnamoner what do you want to talk about?

**Comment by cinnamoner** [comment]
@t-oster Thanks for jumping in. I would like to know whether it is or not possible to arrange desktop icons from the contextual menu as in Windows and/or Plasma?

**Comment by t-oster** [comment]
hi, I am not a cinnamon developer, so I have no idea. However since you can sort them right now via context menu, there should not be much to do in order to just arrange them. The main difference between the arrangement in windows and cinnamon is just, that in windows all icons have the same height per default, which results in a regular grid, while in cinnamon due to the preview etc the height is different, which causes only the columns to be equal width, but not the rows to be equal height.

**Comment by cinnamoner** [comment]
@t-oster Alright, thanks.

**Comment by cinnamoner** [comment]
@feren seems Joe deleted your post as well. What were you saying? Say it again right here, please.

**Comment by lestcape** [comment]
@t-oster and what about if the icons will be re-sized to the same height preserving the aspect radio and then put there in a regular box, centring the image inside the box, and then make the walk with the box and not with the icons?

**Comment by lestcape** [comment]
I think most of it it' s done, just the icons are not re-sized... And well some icons could also have a wrong width like the amule icon. If you try to show it inside the desktop.

**Comment by cinnamoner** [comment]
👀

**Comment by cinnamoner** [comment]
![demo](https://cloud.githubusercontent.com/assets/15638605/23589879/7df4581c-01de-11e7-90d1-f24c31f91e93.gif)
Credit: lotsrc


Not bad at all. Like it.

**Comment by cinnamoner** [comment]
@pepa65 You're welcome. Any insights?

**Comment by pepa65** [comment]
Not really, I don't use Nemo, I use Caja.

**Comment by cinnamoner** [comment]
@pepa65 

 > I use Caja

That's alright. Say, have you noticed any glow-up/light-up of folders in Caja when you hover them?

**Comment by pepa65** [comment]
Yes, the icons light up when hovered over, the shortened icon text gets expanded to full length and when overlapping (!) with other icons, get brought forward. No complaints here, I find the highlighting handy and the icon text behaviour a must.

**Comment by cinnamoner** [comment]
> Yes, the icons light up when hovered over

@pepa65 Thanks.

@Elbullazul So, it seems Thunar (Xfce) is the only exception. I've tried AnemOS like a while ago and didn't notice any glow-up there. What's your take on it? I think your W10 is highly suitable for Xfce. Crossed my mind to fork whisker to look like a W10 menu. Just a thought though.

**Comment by feren** [comment]
@cinnamoner Sorry to crush your dreams, but Thunar DOES have a brightening when hovering over folders and files over here...

**Comment by cinnamoner** [comment]
@feren Don't know. I recall there was no glow in AnemOS. At least the *close window* can be made wider.
***

> Sorry to crush your dreams

None crushed.

**Comment by cinnamoner** [comment]
> Thunar DOES have a brightening when hovering over folders and files over here... (It's just more suttle)

@feren How do you know that?

**Comment by feren** [comment]
@cinnamoner Because I just installed it to test that...

**Comment by cinnamoner** [comment]
> Because I just installed it to test that...

@feren Pure-blooded Xfce? The whole DE? What distro?

**Comment by feren** [comment]
@cinnamoner Just Thunar itself in Cinnamon, on feren OS 2017.0 Development

**Comment by cinnamoner** [comment]
> Just Thunar itself in Cinnamon, on feren OS 2017.0 Development

@feren That was my guess. Inconclusive.

**Comment by cinnamoner** [comment]
@feren Anyways, thanks.

**Comment by cinnamoner** [comment]
@Elullazul Only positive feelings are good. What have I troubled you with?

**Comment by cinnamoner** [comment]
Okay. Got business of my own. See you guys.
